apache-rat-tools/src/main/java/org/apache/rat/tools/xsd/XsdGenerator.java

```diff
@@ -73,10 +68,8 @@ public class XsdGenerator {
      * @throws IOException on IO errors.
      * @throws TransformerException if the XSD can not be pretty printed.
      */
-    public static void main(String[] args) throws IOException, TransformerException {
-        ReportConfiguration configuration = OptionCollection.parseCommands(args, (options) -> {
-        });
-        XsdGenerator generator = new XsdGenerator(configuration);
+    public static void main(final String[] args) throws IOException, TransformerException {
```

Comment:
@Claudenw is it correct to remove the configuration here? It seems to be unused.